stingray/lightcurve.py

```diff
@@ -194,6 +202,22 @@ def __init__(self, time, counts, err=None, input_counts=True,
                                               self.tstart + self.tseg]]))
         check_gtis(self.gti)
 
+    def change_mjdref(self, new_mjdref):
```

Comment:
Eek, clearly I didn't read this as carefully as I should have. Sorry about that! I think if it returns a new `Lightcurve` object, this is fine exactly as it is.